### DISCUSSION TITLE
What is the difference between Demucs and Moises models for guitar isolation ?

### DISCUSSION THREAD

**Discussion by JohnnyRico140** [CarlGao4/Demucs-Gui#80] [General]
Hello, I spent A LOT of time comparing all the stem separation solutions until now, and I want to know why Moises Pro can separate guitars from keys so well.

I used famous David Gilmour guitar solo on Pink Floyd Another Brick In The Wall, Part 2 song for comparison and Moises Pro (with keys and guitar separation option) renders guitar so much better to Demucs 6s one...

I’m not an expert when dealing with deep learning and such, I just want to know !

Thanks 

I could upload different renderings, but I think that would not be ok, relative to copyright etc...

**Comment by CarlGao4** [comment]
As for guitar separation, we need to find good enough dataset to train the model. That means, a set of separated songs for the model to learn from

**Reply by JohnnyRico140** [reply]
If I understand well, Moises train his own custom models with maybe more guitar orientend songs, that's why I found it the best  until now ?

**Reply by CarlGao4** [reply]
Yes. Most public dataset for source-separation has only four stems: bass, drums, vocals, other. That's why demucs official models (except htdemucs_6s, which uses private dataset of Meta) only hav these stems. A model with more stems or higher quality can also be trained, as long as we have enough songs to be used.

**Reply by JohnnyRico140** [reply]
Ok, thank you, I was wondering all of this time why Moises was working so well, they have this pro version with 6 stems separation and this is their economic business for training their models more and more, thanks again !

**Reply by anprdev** [reply]
Enough Songs.
I wondering what songs this has to be Flac, Wav or Mp3.
I have about 11k of MP3 and wondering if this can be used.
Also maybe I can help with Train somehow.
Just let me know.
